crates/rpc/types/src/types.rs

```diff
@@ -220,6 +222,27 @@ pub struct RawBlockWitness {
     pub raw_chain_state: Vec<u8>,
 }
 
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct RpcCheckpointCommitmentInfo {
+    /// block where checkpoint was posted
+    pub blockhash: L1BlockId,
+    /// txid of txn for this checkpoint
+    pub txid: Txid,
+    /// wtxid of txn for this checkpoint
+    pub wtxid: Wtxid,
+    // other info
+}
```

Comment:
For convenience could we also include the L1 block height and tx position too?  It's more than just a "commitment" as it is, it's like a "locator" or "selector" or "ref".